manage/deploying/front-end/nginx.rst

```diff
@@ -79,40 +81,42 @@ Then enable the site by creating a symbolic link::
     cd /etc/nginx/sites-enabled
     ln -s ../sites-available/yoursite.conf .
 
-See that your nginx configuration is valid::
+Verify that your nginx configuration is valid::
 
     /etc/init.d/nginx configtest
 
     ok
     configuration file /etc/nginx/nginx.conf test is successful
     nginx.
 
-Alternatively your system might not provide ``configtest`` command and then
-you can test config with::
+If your system does not provide the ``configtest`` command, then you can test the configuration with::
 
     /usr/sbin/nginx
 
-If the config was OK then restart::
+If the configuration is OK, then restart nginx::
 
     /etc/init.d/nginx restart
 
 More info:
 
-* http://wiki.mediatemple.net/w/%28ve%29:Configure_virtual_hosts_with_Nginx_on_Ubuntu
+* https://mediatemple.net/community/products/developer/204405534/install-nginx-on-ubuntu#3
 
-* http://www.starzel.de/blog/securing-plone-sites-with-https-and-nginx
+* https://www.starzel.de/blog/securing-plone-sites-with-https-and-nginx
 
 Content Security Policy (CSP) prevents a wide range of attacks, including cross-site scripting and other cross-site injections, but
 the CSP header setting may require careful tuning.
-To enable it, replace the Content-Security-Policy-Report-Only by Content-Security-Policy.
-The example above works with Plone 4.x and up (including TinyMCE) but it very wide. You may need to adjust it if you want to make CSP more restrictive or use additional Plone Products. For more information, see
+To enable it, replace the ``Content-Security-Policy-Report-Only`` by ``Content-Security-Policy``.
+The example above works with Plone 4.x and up (including TinyMCE) but it is very general.
+You may need to adjust it if you want to make CSP more restrictive or use additional Plone Products.
+For more information, see:
```

Comment:
For this one, I just reviewed grammar, markup, and line breaks. I avoided technical changes because I'm not familiar with them or how to test.

Removed "very".